crates/holochain_websocket/benches/bench.rs

```diff
@@ -25,8 +20,8 @@ fn simple_bench(bench: &mut Criterion) {
 
     let runtime = rt();
 
-    let (listener, listener_address, jh) = runtime.block_on(setup());
-    let (mut send, mut recv) = runtime.block_on(setup_client(listener_address));
+    let (listener_address, jh) = runtime.block_on(setup());
```

Comment:
Given the question above about performance of concurrent zome calls, what do these benchmark tests say about the performance of websocket calls compared with previously?

Reply:
@ThetaSinner 

### branch `develop`

```
simple_bench/client/request                        
                        time:   [95.553 µs 95.946 µs 96.404 µs]
Found 15 outliers among 100 measurements (15.00%)
  6 (6.00%) low mild
  4 (4.00%) high mild
  5 (5.00%) high severe
Benchmarking simple_bench/client/signal: Collecting 100 samples in estimated 5.0                                                                                simple_bench/client/signal                        
                        time:   [2.5645 µs 2.5956 µs 2.6255 µs]
Found 10 outliers among 100 measurements (10.00%)
  10 (10.00%) low mild
Benchmarking simple_bench/client/response: Collecting 100 samples in estimated 5                                                                                simple_bench/client/response                        
                        time:   [86.378 µs 87.104 µs 87.905 µs]
```

### branch `new-websockets`

```
simple_bench/client/request                        
                        time:   [51.063 µs 51.137 µs 51.215 µs]
                        change: [-47.057% -46.610% -46.244%] (p = 0.00 < 0.05)
                        Performance has improved.
Found 5 outliers among 100 measurements (5.00%)
  4 (4.00%) high mild
  1 (1.00%) high severe
Benchmarking simple_bench/client/signal: Collecting 100 samples in estimated 5.0                                                                                simple_bench/client/signal                        
                        time:   [3.9308 µs 3.9330 µs 3.9349 µs]
                        change: [+48.117% +49.494% +50.968%] (p = 0.00 < 0.05)
                        Performance has regressed.
Found 4 outliers among 100 measurements (4.00%)
  1 (1.00%) low severe
  1 (1.00%) low mild
  2 (2.00%) high mild
Benchmarking simple_bench/client/response: Collecting 100 samples in estimated 5                                                                                simple_bench/client/response                        
                        time:   [60.422 µs 60.618 µs 60.830 µs]
                        change: [-33.195% -32.498% -31.741%] (p = 0.00 < 0.05)
                        Performance has improved.
Found 16 outliers among 100 measurements (16.00%)
  6 (6.00%) low severe
  6 (6.00%) low mild
  1 (1.00%) high mild
  3 (3.00%) high severe
```

Reply:
Those are very fast benchmarks for the improvement/regression measurements I think. I'm more inclined to trust the `client/request` and `client/response` tests as they run a bit longer and are showing a performance improvement.

Looks good! Thanks :)